applications/StructuralMechanicsApplication/tests/mixed_u_E_test/mixed_u_E_element_test_materials.json

```diff
@@ -0,0 +1,18 @@
+{
+    "properties" : [{
+        "model_part_name" : "Structure.Parts_Solid_body",
+        "properties_id"   : 1,
+        "Material"        : {
+            "constitutive_law" : {
+                "name" : "LinearElasticPlaneStrain2DLaw"
+            },
+            "Variables"        : {
+                "DENSITY"       : 7850.0,
+                "YOUNG_MODULUS" : 206900000000.0,
+                "POISSON_RATIO" : 0.29,
+                "STABILIZATION_FACTOR" : 0.1
```

Comment:
Have you seen my comment https://github.com/KratosMultiphysics/Kratos/pull/12897#discussion_r1923224856?